ci-operator/config/openshift/knative-eventing-sources/openshift-knative-eventing-sources-release-v0.5.0.yaml

```diff
@@ -0,0 +1,135 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: knative-v0.5.0
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+canonical_go_repository: github.com/knative/eventing-sources
+binary_build_commands: make install
+test_binary_build_commands: make test-install
+tests:
+- as: e2e
+  commands: "INTERNAL_REGISTRY=image-registry.openshift-image-registry.svc:5000 ENABLE_ADMISSION_WEBHOOKS=false make test-e2e"
```

Comment:
this is something I see in serving and eventing (core) - doing the same for sources.

It is `  commands: "make test-e2e"` on files for older releases, @mgencur

Reply:
The parameters are not required for eventing-sources. `make test-e2e` is enough.